sormas-e2e-tests/src/test/java/org/sormas/e2etests/entities/services/SymptomService.java

```diff
@@ -60,4 +62,93 @@ public Symptoms buildEditGeneratedSymptoms() {
         .dateOfSymptom(LocalDate.now().minusDays(2))
         .build();
   }
+
+  public Symptoms buildEditGeneratedSymptomsWithNoOptionsAndYesForOtherSymptoms() {
+    return Symptoms.builder()
+        .maximumBodyTemperatureInC("35,2")
+        .sourceOfBodyTemperature("rectal")
+        .chillsOrSweats(NO.toString())
+        .headache(NO.toString())
+        .feelingIll(NO.toString())
+        .musclePain(NO.toString())
+        .fever(NO.toString())
+        .shivering(NO.toString())
+        .acuteRespiratoryDistressSyndrome(NO.toString())
+        .oxygenSaturationLower94(NO.toString())
+        .cough(NO.toString())
+        .pneumoniaClinicalOrRadiologic(NO.toString())
+        .difficultyBreathing(NO.toString())
+        .rapidBreathing(NO.toString())
+        .respiratoryDiseaseVentilation(NO.toString())
+        .runnyNose(NO.toString())
+        .soreThroat(NO.toString())
+        .fastHeartRate(NO.toString())
+        .diarrhea(NO.toString())
+        .nausea(NO.toString())
+        .lossOfSmell(NO.toString())
+        .lossOfTaste(NO.toString())
+        .otherNonHemorrhagicSymptoms(YES.toString())
+        .symptomsComments(UUID.randomUUID().toString())
+        .firstSymptom("Other clinical symptoms")
+        .dateOfSymptom(LocalDate.now().minusDays(2))
+        .build();
+  }
+
+  public Symptoms buildEditGeneratedSymptomsWithNoOptions() {
+    return Symptoms.builder()
+        .maximumBodyTemperatureInC("35,2")
+        .sourceOfBodyTemperature("rectal")
+        .chillsOrSweats(NO.toString())
+        .headache(NO.toString())
+        .feelingIll(NO.toString())
+        .musclePain(NO.toString())
+        .fever(NO.toString())
+        .shivering(NO.toString())
+        .acuteRespiratoryDistressSyndrome(NO.toString())
+        .oxygenSaturationLower94(NO.toString())
+        .cough(NO.toString())
+        .pneumoniaClinicalOrRadiologic(NO.toString())
+        .difficultyBreathing(NO.toString())
+        .rapidBreathing(NO.toString())
+        .respiratoryDiseaseVentilation(NO.toString())
+        .runnyNose(NO.toString())
+        .soreThroat(NO.toString())
+        .fastHeartRate(NO.toString())
+        .diarrhea(NO.toString())
+        .nausea(NO.toString())
+        .lossOfSmell(NO.toString())
+        .lossOfTaste(NO.toString())
+        .otherNonHemorrhagicSymptoms(NO.toString())
+        .symptomsComments(UUID.randomUUID().toString())
+        .build();
+  }
+
+  public Symptoms buildEditGeneratedSymptomsWithUnknownOptions() {
```

Comment:
create one method for this one and buildEditGeneratedSymptomsWithNoOptions(), and just pass the value which should be set to all fields. Its duplicated code.